### ISSUE TITLE
jsonrepair/lib/umd/index.min.js don't exist

### ISSUE THREAD

**Issue by fab-uleuh** [josdejong/jsonrepair#134]
![Uploading Capture d’écran 2024-12-03 à 20.18.10.png…]()

I’m trying to use a library in my React Native project.
I’m encountering an error:
Android Bundling failed 819ms index.js (2669 modules)  
Unable to resolve "jsonrepair" from "lib/json.ts"  

However, when I use the library with Jest, it works.

---

jsonrepair/lib/umd/index.min.js don't exist, this is jsonrepair.min.js instead. I copy paste into my node_module jsonrepair.min.js in index.min.js and it works

```
While trying to resolve module jsonrepair from file app-mobile/lib/json.ts, the package app-mobile/node_modules/jsonrepair/package.json was successfully found. However, this package itself specifies a main module field that could not be resolved (app-mobile/node_modules/jsonrepair/lib/umd/index.min.js`. Indeed, none of these files exist:

  
app-mobile/node_modules/jsonrepair/lib/umd/index.min.js(.web.ts|.ts|.web.tsx|.tsx|.web.mjs|.mjs|.web.js|.js|.web.jsx|.jsx|.web.json|.json|.web.cjs|.cjs|.web.scss|.scss|.web.sass|.sass|.web.css|.css|.web.css|.css)
app- mobile/node_modules/jsonrepair/lib/umd/index.min.js/index(.web.ts|.ts|.web.tsx|.tsx|.web.mjs|.mjs|.web.js|.js|.web.jsx|.jsx|.web.json|.json|.web.cjs|.cjs|.web.scss|.scss|.web.sass|.sass|.web.css|.css|.web.css|.css)
```

**Comment by fab-uleuh** [comment]
<img width="818" alt="Capture d’écran 2024-12-03 à 20 18 10" src="https://github.com/user-attachments/assets/846b4304-1282-47e3-9602-97561b503b62">

**Comment by josdejong** [comment]
Thanks for reporting, I see the link to the browser bundle is broken. Fixed now in `jsonrepair@3.11.1`
